pkg/asset/cluster/gcp/gcp.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 // Metadata converts an install configuration to GCP metadata.
-func Metadata(config *types.InstallConfig) *gcp.Metadata {
+func Metadata(config *types.InstallConfig, ingressUID string) *gcp.Metadata {
 	return &gcp.Metadata{
-		Region:    config.Platform.GCP.Region,
-		ProjectID: config.Platform.GCP.ProjectID,
+		Region:          config.Platform.GCP.Region,
+		ProjectID:       config.Platform.GCP.ProjectID,
+		LoadBalancerUID: ingressUID,
```

Comment:
if the metadata is lost how can user's recreate this..?

Reply:
It'll be in a configmap on the cluster. But from the uninstaller point of view, we can attempt to uninstall without it if it's no longer there.

Reply:
the uninstaller does not have access to kubernetes api.